site/pom.xml

```diff
@@ -125,6 +125,11 @@
             <artifactId>junit-dataprovider</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
do we need that here?

Reply:
If you mean _'do we need this declaration if we have it declared in parent POM already'_, @wojtek-oledzki,  then see comment below about `dependencyManagement` below.

If you're asking whether we need this dependency in Site module at all - I'll defer to @ngta1 :wink:

Reply:
Good question.  Even though it is defined in the parent POM, I tried removing this dependency from the site pom, but the logging to Splunk stops for some reason (for site, cms and audit).  I have however removed the version as per @ziemsky comment below.